cordapp-example/workflows-kotlin/src/main/kotlin/com/example/flow/PaybackFlow.kt

```diff
@@ -88,12 +76,11 @@ object PaybackFlow {
 
             require(!cashStatesAndRefs.isEmpty()) { "No money" }
             require(sum >= iouState.value) { "Not enough money to send payback, have: $sum needed: ${iouState.value}" }
-            require(cashStatesAndRefs.all { it.state.data.creator == bank }) { "cash must be created by a bank" }
 
```

Comment:
Now it would be good that instead of having "who is `bank`" be decided at the time of payback, that it be decided at the creation time of the IOU.

Reply:
imho its better to decide at paypack, because its more flexible. If I save bank in IOU and there will be new bank available, then I will not be able to accept money from this new one.